app/src/main/java/ch/epfl/sdp/contamination/fragment/InfectionFragment.java

```diff
@@ -28,6 +28,9 @@
 
 import static android.content.Context.BIND_AUTO_CREATE;
 
+/**
+ * Say what this class is doing
```

Comment:
The class is pretty self-explanatory but we should still have an introduction line.
* View displaying the probability of infection of the user

Reply:
done !